### DISCUSSION TITLE
Location of todo.yaml

### DISCUSSION THREAD

**Discussion by zuperman1879** [dooit-org/dooit#184] [General]
Is there a way to change to location of the data file so I can use it on different systems by placing the todo.yaml on a nas ?

**Comment by kraanzu** [comment]
No currently, i use a python library to get the appropriate location right now. Although, have you tried symlinks? Maybe that can help

**Comment by zuperman1879** [comment]
Symlinks .... I never tought of that ... Thanks I'll give it a go
